Cannot run /testsuite anymore - value of org.keycloak.common.Profile.getInstance() is null

### Before reporting an issue

- [x] I have read and understood the above terms for submitting issues, and I understand that my issue may be closed without action if I do not follow them.

### Area

testsuite

### Describe the bug

```
mvn -pl testsuite/integration-arquillian/tests/base clean test -Dtest='org.keycloak.testsuite.oid4vc.**'

Caused by: java.lang.RuntimeException: java.lang.NullPointerException: Cannot invoke "org.keycloak.common.Profile.logUnsupportedFeatures()" because the return value of "org.keycloak.common.Profile.getInstance()" is null
	at org.keycloak.services.resteasy.ResteasyKeycloakApplication.exit(ResteasyKeycloakApplication.java:82)
	at org.keycloak.services.resources.KeycloakApplication.<init>(KeycloakApplication.java:57)
	at org.keycloak.services.resteasy.ResteasyKeycloakApplication.<init>(ResteasyKeycloakApplication.java:48)
	at java.base/jdk.internal.reflect.DirectConstructorHandleAccessor.newInstance(DirectConstructorHandleAccessor.java:62)
	at java.base/java.lang.reflect.Constructor.newInstanceWithCaller(Constructor.java:502)
	at java.base/java.lang.reflect.Constructor.newInstance(Constructor.java:486)
	at org.jboss.resteasy.core.ConstructorInjectorImpl.constructOutsideRequest(ConstructorInjectorImpl.java:203)
	... 71 more
Caused by: java.lang.NullPointerException: Cannot invoke "org.keycloak.common.Profile.logUnsupportedFeatures()" because the return value of "org.keycloak.common.Profile.getInstance()" is null
	at org.keycloak.services.resources.KeycloakApplication.startup(KeycloakApplication.java:82)
	at org.keycloak.services.resteasy.ResteasyKeycloakApplication.initAndStart(ResteasyKeycloakApplication.java:111)
	at org.keycloak.services.resources.KeycloakApplication.<init>(KeycloakApplication.java:55)
	... 76 more
```

This works 

```
mvn -pl testsuite/integration-arquillian/tests/base clean test -Pauth-server-quarkus -Dtest='org.keycloak.testsuite.oid4vc.**'
```

i.e. it only works on quarkus which I cannot use in the IDE to debug stuff

### Version

latest

### Regression

- [x] The issue is a regression

### Expected behavior

should be able to run (old) testsuite on cmd line and IDE (IntelliJ)

### Actual behavior

--

### How to Reproduce?

--

### Anything else?

_No response_

--- a/services/src/test/java/org/keycloak/services/resteasy/ResteasyKeycloakApplication.java
+++ b/services/src/test/java/org/keycloak/services/resteasy/ResteasyKeycloakApplication.java
@@ -46,10 +46,6 @@ public class ResteasyKeycloakApplication extends KeycloakApplication {
     protected Set<Class<?>> classes = new HashSet<>();
 
     public ResteasyKeycloakApplication() {
-        Profile.configure(
-                new PropertiesProfileConfigResolver(System.getProperties()),
-                new PropertiesFileProfileConfigResolver()
-        );
         classes.add(RealmsResource.class);
         if (Profile.isFeatureEnabled(Profile.Feature.ADMIN_API)) {
             classes.add(AdminRoot.class);
@@ -108,6 +104,10 @@ public class ResteasyKeycloakApplication extends KeycloakApplication {
     protected void initAndStart() {
         Config.init(new JsonConfigProviderFactory().create()
                 .orElseThrow(() -> new RuntimeException("Failed to load Keycloak configuration")));
+        Profile.configure(
+                new PropertiesProfileConfigResolver(System.getProperties()),
+                new PropertiesFileProfileConfigResolver()
+        );
         startup();
     }
 

--- a/services/src/test/java/org/keycloak/services/resteasy/ResteasyKeycloakApplication.java
+++ b/services/src/test/java/org/keycloak/services/resteasy/ResteasyKeycloakApplication.java
@@ -24,8 +24,10 @@ import org.keycloak.Config;
 import org.keycloak.common.Profile;
 import org.keycloak.common.profile.PropertiesProfileConfigResolver;
 import org.keycloak.common.util.MultiSiteUtils;
+import org.keycloak.exportimport.ExportImportManager;
 import org.keycloak.models.KeycloakSession;
 import org.keycloak.models.KeycloakSessionFactory;
+import org.keycloak.models.utils.KeycloakModelUtils;
 import org.keycloak.services.error.KcUnrecognizedPropertyExceptionHandler;
 import org.keycloak.services.error.KeycloakErrorHandler;
 import org.keycloak.services.error.KeycloakMismatchedInputExceptionHandler;
@@ -109,6 +111,12 @@ public class ResteasyKeycloakApplication extends KeycloakApplication {
                 new PropertiesFileProfileConfigResolver()
         );
         startup();
+    }
+
+    @Override
+    protected ExportImportManager bootstrap(KeycloakSession session) {
+        // created a nested transaction as this triggers DefaultJpaConnectionProviderFactory.lazyInit
+        return KeycloakModelUtils.runJobInTransactionWithResult(getSessionFactory(), super::bootstrap);
     }
 
 }

